Pipfile

```diff
@@ -40,6 +40,7 @@ simple-websocket = "==0.3"
 sqlalchemy = "==1.4"
 toml = "==0.10"
 tzlocal = "==3.0"
+mkdocs-render-swagger-plugin = "*"
```

Comment:
So I suspect that the Pipfile should be updated as well right?

Reply:
Yes because this is required for the rendering Swagger UI.